models/synthetix/optimism/synthetix_optimism_trades.sql

```diff
@@ -85,20 +93,20 @@ perps AS (
 		,s.evt_tx_hash AS tx_hash
 		,s.evt_index
 	FROM {{ source('synthetix_optimism', 'FuturesMarket_evt_PositionModified') }} AS s
-	LEFT JOIN {{ source('synthetix_optimism', 'FuturesMarketManager_evt_MarketAdded') }} AS sm
+	LEFT JOIN synthetix_markets AS sm
 		ON s.contract_address = sm.market
 	LEFT JOIN asset_price AS p
 		ON s.contract_address = p.market_address
 		AND s.evt_block_time = p.evt_block_time
-	WHERE s.tradeSize != 0
+	WHERE CAST(s.tradeSize AS DOUBLE) != 0
```

Comment:
good catch @RplusT! -- decoded source tables with string datatype on numeric fields is something we are keeping an open conversation on internally. for the time being, when using comparison operators, continue to use a cast or put the value in single quotes

Reply:
@jeff-dude got it!